File: PRAKTIKUM-PEMROGRAMAN1/MODUL4/C/PRAK404-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>

int main () {
    int program;
    float n1, n2;

    do {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan pilihan : ");
        scanf("%d", &program);

        if (program > 0 && program < 5) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &n1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &n2);

            switch (program) {
            case 1:
                printf("Hasil penjumlahan antara %.2f dan %.2f adalah %.2f\n", n1, n2, n1+n2);
                break;
            
            case 2:
                printf("Hasil pengurangan antara %.2f dan %.2f adalah %.2f\n", n1, n2, n1-n2);
                break;

            case 3:
                printf("Hasil perkalian antara %.2f dan %.2f adalah %.2f\n", n1, n2, n1*n2);
                break;

            case 4:
                if (n2 != 0) {
                    printf("Hasil pembagian antara %.2f dan %.2f adalah %.2f\n", n1, n2, n1/n2);
                } else {
                    printf("Pembagian tidak dapat dilakukan.\n");
                }
                break;
            }
        } else if (program != 5) {
            printf("Input Anda salah, silahkan coba lagi.\n");
        }
    } while (program != 5);
    printf ("Terima kasih telah menggunakan kalkulator Ibnu\n");

    return 0;
}

File: PRAKTIKUM-PEMROGRAMAN1/MODUL4/Python/PRAK404-2410817210024-MuhammadIbnuSina.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    program = int(input("Masukkan pilihan: "))

    if 1 <= program <= 4:
        n1 = float(input("Masukkan nilai pertama: "))
        n2 = float(input("Masukkan nilai kedua: "))

        if program == 1:
            print(f"Hasil penjumlahan antara {n1:.2f} dan {n2:.2f} adalah {n1 + n2:.2f}")
        elif program == 2:
            print(f"Hasil pengurangan antara {n1:.2f} dan {n2:.2f} adalah {n1 - n2:.2f}")
        elif program == 3:
            print(f"Hasil perkalian antara {n1:.2f} dan {n2:.2f} adalah {n1 * n2:.2f}")
        elif program == 4:
            if n2 != 0:
                print(f"Hasil pembagian antara {n1:.2f} dan {n2:.2f} adalah {n1 / n2:.2f}")
            else:
                print("Pembagian tidak dapat dilakukan.")
    elif program == 5:
        print("Terima kasih telah menggunakan kalkulator Ibnu")
        break
    else:
        print("Input Anda salah, silakan coba lagi.")